README.md

```diff
@@ -149,6 +149,14 @@ dictionary object constructed from the returned JSON data.
 Returns a Python list of all the available resources for the current
 account.
 
+#### `whoami()`
```

Comment:
This is where you should add the comment. Something like this would work:
```
_Note: This method requires Conjur v1.9+_
```

or something similar to it